docs/docs/api/events.md

```diff
@@ -169,3 +169,20 @@ On Android, the volume may also be lowered on an transient interruption without
 | --------- | --------- | -------------------------------------------- |
 | paused    | `boolean` | On Android when `true` the player should pause playback, when `false` the player may resume playback. On iOS when `true` the playback was paused and when `false` the player may resume playback. |
 | permanent | `boolean` | Whether the interruption is permanent. On Android the player should stop playback.  |
+
+Example (pauses when another app takes over):
+```js
+  // service.js
+  let wasPausedByDuck = false;
+  TrackPlayer.addEventListener(Event.RemoteDuck, async (event) => {
+    const state = await TrackPlayer.getState();
+    if (event.paused && state === State.Playing) {
```

Comment:
@hvihvi Thanks for the contribution!

Question: I've not worked with `RemoteDuck` before, however reading the documentation it looks like it behaves slightly differently on iOS vs Android. Have you taken that into account here?

https://react-native-track-player.js.org/docs/api/events#remoteduck

Another thought is that this could probably also be implemented directly in the `example` project too.

Reply:
My bad, the example project already provides an even better implementation, I should have looked there first. 😅

I amended the commit to link to the example's implementation instead if that helps, but it would also make sense to drop it and leave the doc as it is.